deliverables/README.md

```diff
@@ -2,13 +2,17 @@
 #### Required
 ##### Questions
 - What is a framework?
+ Like many web frameworks, Ruby on Rails uses the model–view–controller (MVC) pattern to organize application programming. In a  default configuration, a model in the Ruby on Rails framework maps to a table in a database, and a Ruby file.
 - What's the difference between a programming language and a framework?
+ A language is syntax, grammar, semantics.. that implementers are required to support. A framework is a cohesive set of -library  code that together simplifies programming in any given language.
```

Comment:
Not just "programming", but it simplifies "app development", by handling the boilerplate, repeatable stuff that is the same across every application.